setup.py

```diff
@@ -105,7 +103,7 @@
     setup_requires=[
         'setuptools_scm',
     ],
-    python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*',
+    python_requires='>=3.4',
```

Comment:
Maybe start at 3.5?

Reply:
Sure. I'll do that as a separate commit, unrelated to this PR, but in the same release.